lib/Language/Souffle/Interpreted.hs

```diff
@@ -10,13 +10,15 @@ module Language.Souffle.Interpreted
   , MonadSouffle(..)
   , SouffleM
   , runSouffle
+  , runSouffleWith
   , cleanup
   ) where
 
 import Prelude hiding (init)
 
 import Control.DeepSeq (deepseq)
 import Control.Monad.State.Strict
+import Control.Monad.Trans.Reader
```

Comment:
`Control.Monad.Reader` avoids the `lift`s below and avoids having to add `transformers` to the dependencies.